### DISCUSSION TITLE
Problem with rsync pulses

### DISCUSSION THREAD

**Discussion by DMeliq** [pyControl/code#76] [Q&A]
Hi everyone,

I am using pyControl in combination with openephys and neuropixel probes, with the microcontroller sending sync pulses to the SMA connector on the NPX board. I have a strange issue in some of the session recorded, where the TTL timestamps periodically reset their values, or show a value of -1 (often this happens before the reset, but not always).  The mean IPI in the rsync function was set to 1000 in this experiment, and this are the resulting timestamps:
![image](https://user-images.githubusercontent.com/106309287/225875353-946f8627-4196-42c7-995d-7a0ce3bd4264.png)

Any idea of what could cause this?

**Comment by ThomasAkam** [comment]
Hi @DMeliq,

If I understand correctly this issue is with the data recorded by the Neuropixels system, not that generated by pyControl, so it would probably be best to ask neuropixels support for help.

From the data in your post it does look like the sample numbers are increasing monotonically as expected so potentially you can use those for the synchronisation.
